SafeguardSessions/modules/error/SchemaTransformationErrors.pqm

```diff
@@ -0,0 +1,8 @@
+let
+    ErrorBase = Extension.ImportModule("ErrorBase.pqm"),
+    SchemaTransformationErrors.ListToTextConversionError = (detail as record) =>
+        ErrorBase("Not A Text List", "The list contains other types than text.", detail)

```

Comment:
I know it is untold, but if you check, the reason field in all of our custom errors contains the error type in `Title Case`.

So I think we should follow the pattern and `Not A Text List` should be `List to Text Conversion Error`.

Reply:
My bad. I missed it.

Reply:
Done!